data/src/main/java/com/microsoft/azure/kusto/data/auth/ConnectionStringBuilder.java

```diff
@@ -164,12 +199,16 @@ public ConnectionStringBuilder(ConnectionStringBuilder other) {
         this.userNameForTracing = other.userNameForTracing;
         this.appendedClientVersionForTracing = other.appendedClientVersionForTracing;
         this.applicationNameForTracing = other.applicationNameForTracing;
+        this.sendX509 = other.sendX509;
+        this.initialCatalog = other.initialCatalog;
```

Comment:
seems like you are not really using all the new parameters - is this a next PR ?